src/tool/UIE/PropertyView.cpp

```diff
@@ -107,19 +107,19 @@ void CPropertyView::OnInitialUpdate() {
         m_UIBase.AddPropItem("Tooltip text", "", PIT_EDIT, "");   // m_pszTooltipText
         m_UIBase.AddPropItem("Open sound", "", PIT_FILE, "");     // sound
         m_UIBase.AddPropItem("Close sound", "", PIT_FILE, "");    // sound
-        m_UIBase.AddPropItem("Delete sound", "", PIT_BUTTON, ""); // sound ��� ����
+        m_UIBase.AddPropItem("Delete sound", "", PIT_BUTTON, ""); // sound ���� �����
         m_UIBase.AddPropItem("Visible", "", PIT_BUTTON, "");      // visible
 
         // image
         m_UIImage.AddPropItem("Texture", "", PIT_FILE, "");   // texture
-        m_UIImage.AddPropItem("UV left", "", PIT_BUTTON, ""); // m_frcUVRect (UV��ǥ)
+        m_UIImage.AddPropItem("UV left", "", PIT_BUTTON, ""); // m_frcUVRect (UV��ǥ)
         m_UIImage.AddPropItem("UV top", "", PIT_BUTTON, "");
         m_UIImage.AddPropItem("UV right", "", PIT_BUTTON, "");
         m_UIImage.AddPropItem("UV bottom", "", PIT_BUTTON, "");
         m_UIImage.AddPropItem("Color", "", PIT_COLOR, ""); // m_Color
         m_UIImage.AddPropItem("Animate Frame", "", PIT_EDIT,
-                              ""); // Animate frame �ʴ� �� �������� ���ϸ���Ʈ �� ���ΰ�?
-        m_UIImage.AddPropItem("Make Animation", "", PIT_BUTTON, ""); // Animation��� �����
+                              ""); // Animate frame �ʴ� �� ���������� ���ϸ���Ʈ �� ���ΰ�?
+        m_UIImage.AddPropItem("Make Animation", "", PIT_BUTTON, ""); // Animation���� �����
```

Comment:
@SeniourMarquies, could you please fix the git diffs? Your text editor broke the encoding of this file, which made all Korean characters to become junk data.

Reply:
@stevewgr I did some updates on encoding.